primeqa/ir/README.md

```diff
@@ -150,12 +150,12 @@ Using a model trained as described [here](https://github.com/primeqa/primeqa/tre
 python primeqa/ir/run_ir.py \
     --engine_type DPR \
     --do_index \
-    --dpr_ctx_encoder_path <context_encoder_model> \
+    --ctx_encoder_name_or_path <context_encoder_model> \
```

Comment:
Can we please make this arg be `model_name_or_path` to be consistent?

Reply:
Left alone as after a separate discussion.